aldryn_jobs/south_migrations/0035_auto__del_unique_jobsconfig_type_namespace.py

```diff
@@ -10,11 +10,15 @@ class Migration(SchemaMigration):
     def forwards(self, orm):
         # Removing unique constraint on 'JobsConfig', fields ['type', 'namespace']
         db.delete_unique(u'aldryn_jobs_jobsconfig', ['type', 'namespace'])
```

Comment:
Because we allow openings with the same slug when they're defined on different categories or apphooks.